src/Blendedness.cc

```diff
@@ -197,41 +198,59 @@ BlendednessAlgorithm::BlendednessAlgorithm(Control const& ctrl, std::string cons
     if (_ctrl.doOld) {
         _old = schema.addField<double>(
                 schema.join(name, "old"),
-                "blendedness from dot products: (child.dot(parent)/child.dot(child) - 1)");
+                "Blendedness from dot products: (child.dot(parent)/child.dot(child) - 1)");
     }
     if (_ctrl.doFlux) {
         _instFluxRaw = schema.addField<double>(
-                schema.join(name, "raw_instFlux"),
-                "measure of how instFlux is affected by neighbors: (1 - instFlux.child/instFlux.parent)");
+                schema.join(name, "raw"),
+                "Measure of how much instFlux is affected by neighbors: "
+                "(1 - child_instFlux/parent_instFlux).  Operates on the \"raw\" pixel values.");
         _instFluxChildRaw = schema.addField<double>(
-                schema.join(name, "raw_instFlux_child"),
-                "instFlux of the child, measured with a Gaussian weight matched to the child", "count");
+                schema.join(name, "raw_child_instFlux"),
+                "instFlux of the child, measured with a Gaussian weight matched to the child.  "
```

Comment:
I'd spell out "Instrumental flux" instead of "instFlux" in the docstrings for this and other fields that end in `_instFlux`.